### PR TITLE
Feature/dragos testing

### PR DESCRIPTION
**Pull request by dragospopa**
Solved the issues mentioned during the code review.

### CODE REVIEW

File: transformer/docker_compose_service.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	ExternalLinks Links
+	ExtraHosts    ExtraHosts
+	Isolation     Isolation
+	Networks      Networks
+	Pid           ExtraHosts
+	Secrets       Secrets
+	SecurityOpt   SecurityOpt
+	UsernsMode    UsernsMode
+	Ulimits       Ulimits
+	Healthcheck   Healthcheck
```
Comment (DimitriosLisenko) [inline transformer/docker_compose_service.go]: Also add CPUs as one of the unsupported

File: transformer/utils.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		}
+	}
+	return false
+}
+
+func checkDB(image string) (string, bool) {
+	db_check := []string{"mysql", "postgresql", "redis", "mongodb", "elasticsearch", "glusterfs", "influxdb", "rabbitmq", "sqlite", "postgres", "mongo", "influx"}
+
+	for i := 0; i < len(db_check); i++ {
+		if strings.Contains(image, db_check[i]) || strings.Contains(db_check[i], image) {
```
Comment (DimitriosLisenko) [inline transformer/utils.go]: Just check if image contains any of db_check

File: transformer/utils.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+			}
+		}
+	}
+	return "", false
+}
+
+func CheckError(err error) {
+	if err != nil {
+		fmt.Println(err.Error())
+		os.Exit(0)
```
Comment (DimitriosLisenko) [inline transformer/utils.go]: 0 is a successful exist - use os.Exit(1)

File: transformer/transform.go [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+		serviceYaml.Services, serviceYaml.Dbs = copyToServiceYML(d)
+
+	} else {
+		serviceYaml.Services, serviceYaml.Dbs = copyToServiceYML(dockerCompose.Services)
+	}
+
+	file, err := yaml.Marshal(serviceYaml)
+
+	file = []byte("# Generated with <3 by Cloud66\n\n"+string(file))
```
Comment (DimitriosLisenko) [inline transformer/transform.go]: @khash Is <3 ok in this company? :D

File: transformer/custom_yaml_unmarshler.go [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+			return err
+		}
+		e.BuildCommand = single
+	} else {
+		e.Build.Dockerfile = build.Dockerfile
+	}
+	return nil
+}
+
+func (sm *Volumes) UnmarshalYAML(unmarshal func(interface{}) error) error {
```
Comment (dragospopa) [inline transformer/custom_yaml_unmarshler.go]: Solved 👍

File: transformer/utils.go (hunk 3 above)
Comment (dragospopa) [inline transformer/utils.go]: Solved 👍

File: transformer/utils.go (hunk 2 above)
Comment (dragospopa) [inline transformer/utils.go]: Solved 👍

File: transformer/docker_compose_service.go (hunk 1 above)
Comment (dragospopa) [inline transformer/docker_compose_service.go]: Solved 👍

**Pull request merged**